src/i18n/strings/en_US.json

```diff
@@ -134,6 +134,8 @@
     "Failed to join room": "Failed to join room",
     "Failed to kick": "Failed to kick",
     "Failed to leave room": "Failed to leave room",
+    "Failed to load %(groupId)s": "Failed to load %(groupId)s",
+    "Failed to load group members": "Failed to load group members",
```

Comment:
ftr, you don't need to add entries to this file: en_US falls back to en_EN automatically (even if there is a US english translation, it's generally better to add it in weblate otherwise you'll get conflicts in weblate).